operators/src/source/gdal_source.rs

```diff
@@ -694,29 +700,28 @@ where
         info: GdalLoadingInfoPart,
     ) -> impl Stream<Item = Result<RasterTile2D<T>>> {
         let spatial_resolution = query.spatial_resolution;
-        let geo_transform = info.params.geo_transform;
-
-        // adjust the spatial resolution to the sign of the geotransform
-        let x_signed = if geo_transform.x_pixel_size.is_sign_positive()
-            && spatial_resolution.x.is_sign_positive()
-        {
-            spatial_resolution.x
+        let no_data_value = self.no_data_value;
+
+        // TODO: re-evaluate the logic behind this
+        // The "Pixel-space" starts at the top-left corner of a GDAL image.
+        // We assume that GDAL datasets are not flipped.
+        // Therefore, the x-axis is always increasing
+        debug_assert!(spatial_resolution.x.is_sign_positive());
+        // and the y-axis is always decreasing
+        let spatial_resolution_y = if spatial_resolution.y.is_sign_positive() {
+            spatial_resolution.y * -1.0
         } else {
-            spatial_resolution.x * -1.0
-        };
-
-        let y_signed = if geo_transform.y_pixel_size.is_sign_positive()
-            && spatial_resolution.y.is_sign_positive()
-        {
             spatial_resolution.y
-        } else {
-            spatial_resolution.y * -1.0
         };
 
-        let tiling_strategy = self.tiling_specification.strategy(x_signed, y_signed);
+        let tiling_strategy = self
```

Comment:
can we enforce/check that there are no temporal gaps in between two consecutive spatial tile streams? Maybe when turning the meta_data loading_info into tile_streams?

Reply:
i say yes but we need more information. We  can add the SparseTilesFillAdapter to fill missing  spatial Tiles. We would need something similar to add Tiles in temporal holes. Do you want me to do  that in this  task?